content/docs/user-guide/experiment-management/index.md

```diff
@@ -1,110 +0,0 @@
-# Experiment Management
-
-Data science and machine learning are iterative processes that require a large
-number of attempts to reach a certain level of a metric. Experimentation is part
-of the development of data features, hyperspace exploration, deep learning
-optimization, etc.
```

Comment:
> I dropped this page...

How about using the index pago for what's left in overview.md though? Instead of having an extra "Overview" nav entry.